### ISSUE TITLE
Add instructions on how to access the CLI

### ISSUE THREAD

**Issue by lwandzura** [influxdata/docs-v2#4041]
Customer was not sure how to install the CLI.  I believe they mean "access the CLI".  They were reading this doc "influx - InfluxDB command line interface":
https://docs.influxdata.com/influxdb/v1.8/tools/influx-cli/

The instructions on how to use the CLI are here:

https://docs.influxdata.com/influxdb/v1.8/tools/shell/#launch-influx

To access the CLI, first launch the influxd database process and then launch influx in your terminal. Once you’ve entered the shell and successfully connected to an InfluxDB node, you’ll see the following output:
$ influx
Connected to http://localhost:8086 version 1.8.10
InfluxDB shell version: 1.8.10
[Change InfluxDB URL](https://docs.influxdata.com/influxdb/v1.8/tools/shell/#)
Note: The versions of InfluxDB and the CLI should be identical. If not, parsing issues can occur with queries.

Add a link to the doc Using influx - InfluxDB command line interface

**Comment by sanderson** [comment]
Yeah, I think we just need to callout at the `influx` CLI is included in the InfluxDB package. If they installed InfluxDB using a package manager, it should be installed on their $PATH.

**Comment by lwandzura** [comment]
100% agree :-)
